Configuration/AlCa/python/autoAlca.py

```diff
@@ -2,7 +2,7 @@
                   'Charmonium'     : 'TkAlJpsiMuMu',
                   'Commissioning'  : 'HcalCalIsoTrk',
                   'Cosmics'        : 'TkAlCosmics0T+MuAlGlobalCosmics+HcalCalHOCosmics+DtCalibCosmics',
-                  'DoubleEG'       : 'EcalCalZElectron+EcalUncalZElectron+HcalCalIterativePhiSym+HcalCalIsoTrkFilter',
+                  'DoubleEG'       : 'EcalCalZElectron+EcalUncalZElectron+EcalTrg+HcalCalIterativePhiSym+HcalCalIsoTrkFilter',
```

Comment:
Something like:
https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/Calibration/EcalAlCaRecoProducers/python/alcastreamEcalEtaCalib_cff.py#L7

ideally coupling the eventSetupPathKey to some HLT path or L1 bit you want to skim on.